setup/cli-install.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * CLI install script for MODX
+ *
+ * Run and answer the questions. Also you can pre-define values:
+ * php www/setup/install.php --database=my_db --database_user=db_user_12 --language=ru
+ */
+
+if (PHP_SAPI != 'cli') {
+    exit('This file should be run through the command line interface!');
+}
+$path = dirname(__FILE__) . '/';
+$config = $path . 'config.xml';
+$mode = 'new';
+
+$variables = [
+    'database_type' => 'mysql',
+    'database_server' => [
+        'prompt' => 'Database server',
+        'default' => '127.0.0.1',
+    ],
+    'database' => [
+        'prompt' => 'Database',
+        'default' => '',
+    ],
+    'database_user' => [
+        'prompt' => 'Database user',
+        'default' => '',
+    ],
+    'database_password' => [
+        'prompt' => 'Database password',
+        'default' => '',
+    ],
+    'database_connection_charset' => 'utf8',
+    'database_charset' => 'utf8',
+    'database_collation' => 'utf8_general_ci',
+    'table_prefix' => [
+        'prompt' => 'Table prefix',
+        'default' => 'modx_',
+    ],
+    'https_port' => 443,
+    'http_host' => [
+        'prompt' => 'Http host',
+        'default' => '',
+    ],
+    'cache_disabled' => 0,
+    'inplace' => 0,
+    'unpacked' => 0,
+    'language' => [
+        'prompt' => 'Language',
+        'default' => 'en',
+    ],
+    'cmsadmin' => [
+        'prompt' => 'Admin login',
```

Comment:
Might be nice if we could use the lexicons for this to make sure that the fields are labeled the same in the UI and the CLI helper. For example, this one is "Administrator username" https://github.com/modxcms/revolution/blob/2.x/setup/lang/en/default.inc.php#L41

A simple implementation could just include the lexicon file at the top, and use `$_lang['foo_bar']` in the prompt definitions.

Bonus points for a `--language=nl` on the cli to also ask for values in the localised language (looks like that would already affect the installed language), but I'd consider that a nice to have, not required for this to be accepted.

Reply:
Interesting idea